src/javascript/actions/goBack.action.jsx

```diff
@@ -22,6 +23,20 @@ const GoBack = ({render: Render, componentProps, ...otherProps}) => {
 
     const onCloseDialog = useCallback(() => setOpen(false), [setOpen]);
 
+    useKeydownListener(event => {
+        if (event.keyCode === 27) {
```

Comment:
could we use a constant here keyCodes.close or keyCodes.back?

Reply:
Ah, good catch forgot about this one